source/pasta/brusselSproutGnocchi.rst

```diff
@@ -0,0 +1,48 @@
+Gnocchi with Brussel Sprouts, Garlic, and a boat-load of butter
+===============================================================
+
+Details
+-------
+
+* Number of servings: 6
+* Cook time: 20 minutes
+
+Ingredients
+-----------
+
+* 1 lb Brussel sprouts, trimmed and halved so they have a big flat face
+* 500g gnocchi (or any type of pasta)
+* At least 4 cloves of garlic (prefferably more), finely minced
+* The juice and zest of 1 lemon (zest less important than juice)
+* 6 tbsp butter
+* Chilli flakes
+
+Optional:
```

Comment:
These should be included within the ingredient list. Place a "*(optional)*" before the ingredient in the list. Take a look at the source for some of the other recipes for clarification if needed.